src/NServiceBus.PersistenceTesting/IPersistenceTestsConfiguration.cs

```diff
@@ -34,6 +34,10 @@ public interface IPersistenceTestsConfiguration
 
         SagaMetadataCollection SagaMetadataCollection { get; }
 
+        /// <summary>
+        /// Initialise any properties needed within the Configure method
+        /// </summary>
+        /// <returns></returns>
```

Comment:
```suggestion
```

Reply:
I don't see value in having this one XML comment at all. I think it should be removed entirely.